test/performance/scale_from_zero_test.go

```diff
@@ -160,44 +162,83 @@ func parallelScaleFromZero(t *testing.T, count int) ([]time.Duration, error) {
 	return durations, g.Wait()
 }
 
-func getStats(durations []time.Duration) *stats {
+func getRunStats(durations []time.Duration) *stats {
 	if len(durations) == 0 {
 		return nil
 	}
-	var avg time.Duration
-
-	for _, dur := range durations {
+	var avg, min, max time.Duration
```

Comment:
well, what I mean, is that `var` is no longer required, since you can initialize with `min := ...`, etc.

Reply:
Same below.